docs/redirects.map

```diff
@@ -560,3 +560,5 @@ genindex.html
 tools.html -> /tools/assistant.html
 examples.html -> /examples/bond-trading/index.html
 search.html
+getting-started/installation.htm -> /getting-started/installation.html
+gsg -> /getting-started/installation.html
```

Comment:
Once we have the new getting started guide we should find a more interesting place to link to than the installation page.

Reply:
Do you think it's better to not create the shortlink yet, then? I'll let @bame-da decide.

Reply:
I don’t think it does any harm given that we can change it later.